tests/components/deconz/test_switch.py

```diff
@@ -77,6 +78,7 @@ async def test_switches(hass):
     assert "switch.unsupported_switch" not in gateway.deconz_ids
     assert "switch.on_off_relay" in gateway.deconz_ids
     assert len(hass.states.async_all()) == 5
+    assert len(gateway.entities[switch.DOMAIN]) == 4
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)